### DISCUSSION TITLE
How I can mix ejs with md?

### DISCUSSION THREAD

**Discussion by Phosphorus-M** [hexojs/hexo#4907] [Question / Answer]
Hi! I'm trying to make a custom theme, I'm doing that in ejs but to me it could be easier if I could you this plugin:
https://github.com/sergeyzwezdin/hexo-images
So basically I want to use this md plugin inside of the ejs but I don't sure how, Hexo has a helper to render since ejs?

**Comment by lorezyra** [comment]
If `hexo-images` registers itself as a hexo tag, then you can simply use `{% hexoImageTag /path/to/img %}` in the markdown. (Didn't read that code...) Otherwise, you would have to create a helper that registers the tag for use inside the markdown file. If you require it inside the EJS file, then you would need to use a function call `<%= hexoImageTag("/path/to/img") %>`.
